config/jobs/kubernetes-csi/csi-driver-host-path/csi-driver-host-path.yaml

```diff
@@ -15,10 +15,89 @@ presubmits:
         command:
         - runner.sh
         args:
-        - ./.prow.sh
+        - /bin/sh
+        - -c
+        # Not all branches have Prow support.
+        - if [ -e .prow.sh ]; then ./.prow.sh; fi
+        env:
+        - name: CSI_PROW_KUBERNETES_VERSION
+          value: "1.13.3"
```

Comment:
this will get out of date quickly as more patch releases are created.  @krzyzacy I know that the current CI jobs we have can figure out what is the latest patch release. Can you point us on how we can use that?

Reply:
I don't think we absolutely have to track the latest 1.13.x version here. This is basically our "does the modified component still work on 1.13" test, and as long we pass that for some 1.13.x release I think we are fine.

There's an additional constraint here: in order to bring up the cluster quickly, the job uses the pre-built Kubernetes images for kind. The very latest Kubernetes 1.13.x might not have that image (yet).

Reply:
Unfortunately patch releases could have breaking changes, so it would still be good to test whatever the latest official release is, or maybe even head of the release branch if we want to proactively find regressions in a release branch.

I know there's a file that gets updated with the latest images after they are available, and the current k/k CI parses the file bring up the cluster with those images.

Reply:
A pre-submit job must be fully reproducible. Consider what happens if our stable components work fine with Kubernetes 1.13.3, someone makes a minimal change in the component and suddenly the test no longer passes because it now runs against 1.13.4 and there was a bug in that release. That would be confusing and block all development.

Testing against "the latest" version of a certain branch can be in a periodic job similar to the one I am adding in this PR for master.

Reply:
I've added a second periodic job for testing on Kubernetes 1.13.x. It gets built from source.